modules/data-loading/pages/data-streaming-connector/big-query.adoc

```diff
@@ -0,0 +1,276 @@
+= Querying and Streaming from Google BigQuery
+:description: Steps to query and streaming data from Google BigQuery using Data Streaming Connector.
+:sectnums:
+
+The Data Streaming Connector allows you to invoke SQL queries to your Google BigQuery dataset and stream the query results to TigerGraph’s internal Kafka server with a specified topic.
+You can then create and run a xref:kafka-loader/index.adoc[Kafka loading job] to load data from Kafka into your graphs.
+
+== Prerequisites
+* You should have one of the following authentication credentials:
+** Google Service Account credentials
+** Access and refresh Tokens
+
+== Procedure
+
+=== Specify connector configurations
+
+The connector configurations provide the following information:
+
+* Connector class
+* Connection URL
+* Value converter
+* SQL query statement
+* Connector properties
+
+==== Specify connector class
+The connector class indicates what type of connector the configuration file is used to create.
+Connector class is specified by the `connector.class` key.
+
+For streaming from Google BigQuery, the class is `io.confluent.connect.jdbc.JdbcSourceConnector`.
+
+[source,text]
+----
+connector.class=io.confluent.connect.jdbc.JdbcSourceConnector
+----
+
+==== Specify connection URL
+
+To stream data from BigQuery, Data Streaming Connector uses the https://cloud.google.com/bigquery/docs/reference/odbc-jdbc-drivers#current_jdbc_driver[BigQuery JDBC Driver].
+The connection URL requires the domain name or IP address of the BigQuery server (`\https://www.googleapis.com/bigquery/v2`), the port number, the ID of your BigQuery project, and your credentials for authentication.
+You can also supply other optional properties to the connection that are supported by https://storage.googleapis.com/simba-bq-release/jdbc/Simba%20Google%20BigQuery%20JDBC%20Connector%20Install%20and%20Configuration%20Guide_1.3.0.1001.pdf[the BigQuery JDBC connector]:
+
+[source.wrap,text]
+----
+connection.url=jdbc:bigquery://https://www.googleapis.com/bigquery/v2:<port_number>;<property1>=<value1>;<property2>=<value2>;... <1>
+----
+<1> Each pair of property and value is separated by a semicolon``;``.
+
+The following is a list of required or frequently used properties:
+
+|===
+|Property |Description
+
+|`ProjectId`
+|ID of your Google Cloud Project for you BigQuery server.
+Required.
+
+|`OAuthType`
+a|Required. A number that specifies the type of authentication used by the connector:
+
+* `0`: Authentication with Google Service Account.
+This requires that you supply two additional parameters:
+** `OAuthServiceAcctEmail`
+** `OAuthPvtKeyPath`
+* `1`: Authentication with access token or refresh token.
```

Comment:
sorry, this should be `2`, it's a typo. We support `0` or `2`. `1` is unsupported because it needs to interact with the browser.